library/crates/BUILD.bazel

```diff
@@ -23,7 +23,25 @@ alias(
 
```

Comment:
This file, `crates.bzl`, and everything under the `remote` folder, are auto-generated by `cargo raze`.

All manual changes are in `Cargo.toml`.